docs/operations/hotplug_interfaces.md

```diff
@@ -79,27 +83,70 @@ Please refer to the
 for more info.
 
 Once the virtual machine is running, and the attachment configuration
-provisioned, the user can request the interface hotplug operation. Please refer
-to the following snippet:
-```bash
-virtctl addinterface vm-fedora --network-attachment-definition-name new-fancy-net --name dyniface1
```

Comment:
It may be good to add a note that `virtctl` commands used in v1.0 are dropped.
Not completely sure if it is a good move though.

Reply:
It seems virctl hotplug/unplug commands are available in v1.0.0 and removed in v1.1.0
https://github.com/kubevirt/kubevirt/tree/v1.0.0/pkg/virtctl/network